frontend/app/components/authentication/login.hbs

```diff
@@ -28,21 +33,34 @@
         {{t "messages.authenticating"}}
       </button>
       {{else}}
-      <BsButton class="bg-primary form-control mb-2" @defaultText={{t "auth.login.login" }} type="submit">
+      <BsButton class="bg-primary form-control mb-4" @defaultText={{t "auth.login.login" }} type="submit">
+      </BsButton>
+
+      <p class="line-text"><span>{{t "auth.login.or" }}</span></p>
+
+      <BsButton type="button" class="shadow-sm rounded btn-dark btn-linkedIn form-control mb-2" {{ on "click"
+        this.authenticateWithLinkedIn}}>
```

Comment:
This  getter/property does not exist on component class

Reply:
It actually exists